src/styles/_md-styles.scss

```diff
@@ -20,6 +20,20 @@ example,
   margin-top: 1rem;
   margin-bottom: 3rem;
   display: block;
+
+
+  .decorated {
+    display: flex;
+}
+
+  &--contains-icon {
```

Comment:
Your icons are still rendering extra large for me and I'm not seeing these  `--contains-icon` styles being applied.